cfg/eww/scripts/getSongPosition

```diff
@@ -1,26 +1,35 @@
 #!/bin/sh
 
-# Change this to something above 0 if you have Spotify Premium.
+# Author Notes:
 # Defaults to 0 cause I don't have Premium :(
-HAS_SPOTIFY_PREMIUM=0
-
-print_position() {
-    POSITION=$(playerctl position | sed 's/..\{6\}$//')
-    DURATION=$(playerctl metadata mpris:length | sed 's/.\{6\}$//')
-    printf "%0d:%02d " $((POSITION%3600/60)) $((POSITION%60))
-    printf "/ %0d:%02d" $((DURATION%3600/60)) $((DURATION%60))
-}
+# Change the (has_premium) from line 15, collumm 17 into 1 if you have spotify premium
+# For some reason firefox position doesn't work I probably need to check more players if they support possition
 
 if ps -C spotify > /dev/null; then
     PLAYER="spotify"
+elif ps -C firefox > /dev/null; then
+    PLAYER="firefox"
 fi
 
 if [ "$PLAYER" = "spotify" ]; then
-    if [ $HAS_SPOTIFY_PREMIUM -gt 0 ]; then
-        print_position    
-    fi
+    has_premium=0
+elif [ "$PLAYER" = "firefox" ]; then
+    not_support=1
 else
     if [ $(playerctl metadata mpris:length | wc -c) -gt 0 ]; then
         print_position
     fi
 fi
+
+if [ $has_premium -gt 0 ]; then
+    POSITION=$(playerctl position | sed 's/..\{6\}$//')
+    DURATION=$(playerctl metadata mpris:length | sed 's/.\{6\}$//')
+    printf "%0d:%02d " $((POSITION%3600/60)) $((POSITION%60))
+    printf "/ %0d:%02d" $((DURATION%3600/60)) $((DURATION%60))
+elif [ $not_support -gt 0 ]; then
+    echo "Player does not support possition"
+else
+    echo "You don't have Spotify Premium."
+fi
+
+# IDK maybe fix this I guess
```

Comment:
fixed in recent commits lmao should discard this on the pr

Reply:
yeah my method is kinda hacky